test/software/amazon/ion/SymbolTest.java

```diff
@@ -216,7 +200,9 @@ public void testCloneSymbolWithKnownText()
     @Test
     public void testSymbolWithEscapedNewline()
     {
-        badValue("'\\\n'");
```

Comment:
This used to fail because `\\\n` is an escaped newline, which expands to nothing within quoted text, leaving this as an empty symbol value. We should be testing that that case is now valid (and is treated as an empty symbol). This might actually be a good addition to ion-tests if it's not already there.

`"'\\n'"` is also an interesting test; it's just a different test.